packages/create-subscription/src/__tests__/createSubscription-test.internal.js

```diff
@@ -160,7 +160,7 @@ describe('createSubscription', () => {
 
     it('should still work if unsubscription is managed incorrectly', async () => {
       const Subscription = createSubscription({
-        getCurrentValue: source => undefined,
+        getCurrentValue: source => source.value,
```

Comment:
Is this a breaking change?

Reply:
I looked into this for a while and I think the old version was just wrong and it accidentally worked with the old implementation. I'm going to try to write a failing test for what's in master.

Reply:
Never mind, I found the bug. Will add a test.